intelmq/bots/parsers/shadowserver/parser.py

```diff
@@ -265,5 +266,10 @@ def run(cls, parsed_args=None):
         else:
             super().run(parsed_args=parsed_args)
 
+    def test_update_schema(cls):
```

Comment:
Please add a docstring explaining why this test method is needed in the bot's code.

Reply:
Added.